site/docs/BestPractice-ServingOverHTTP.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Serving over HTTP
+layout: ../_core/DocsLayout
+category: Best Practices
+permalink: /learn/serving-over-http/
+next: /learn/authorization/
+---
+
+HTTP is a great choice for client-server communication when using GraphQL because of its ubiquity. Here are some guidelines for setting up a GraphQL server to operate over HTTP.
+
+### Web Request Pipeline
+Most modern web frameworks use a pipeline model where requests are passed through a stack of middleware (aka filters/plugins). As the request flows through the pipeline, it can be inspected, transformed, modified, or terminated with a response. GraphQL should be placed after all authentication middleware.
+
+### URIs, Routes, and Controllers
+HTTP is commonly associated with REST, which uses "resources" as its core concept. In contrast, GraphQL uses an entity graph as its conceptual model. As a result, entities in GraphQL are not identified by URIs. Instead, a GraphQL server operates on a single URI/endpoint. All GraphQL requests for a given graph/domain should be directed at this endpoint. If you are coming from an MVC framework, think of GraphQL as a unified controller that handles requests for all entities.
```

Comment:
I don't get the line about "think of GraphQL as a unified controller" - that doesn't seem very MVC to me.

You might want to say URL because some people still don't know what URI is.